csharp-basic-topics/GuidClassInCSharp/GuidClassInCSharp/Methods/GuidToByteString.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace GuidClassInCSharp
+{
+    partial class GuidClassInCSharpMethods
+    {
+        public static string GuidToByteString(Guid inputGuid)
+        {
+            var byteArray = inputGuid.ToByteArray();
+
+            return BitConverter.ToString(byteArray);
```

Comment:
This may be fine, if for the purpose of the article you want a hex string separated by dashes, otherwise, since you are targetting .NET 6.0+, you can use the Convert.ToHexString method. I understand that in this case the performance doesn't really matter, but when possible we should try to show best practices even in our sample code.
https://learn.microsoft.com/en-us/dotnet/api/system.convert.tohexstring?view=net-7.0

Reply:
In this specific case, I wanted to show the separate hexadecimal digits and notify that they are a collection of hexadecimal pairs. That said, I preferred "31-A7-5A-C8-0C-24-D5-46-98-A8-ED-98-84-97-F8-03" over "31A75AC80C24D54698A8ED988497F803".

Reply:
👍I thought that may be the case, just commented to be sure that there was a specific goal